tests/coalaHTMLTest.py

```diff
@@ -16,13 +19,23 @@
 class coalaHTMLTest(unittest.TestCase):
 
     def setUp(self):
+        os.chdir(os.path.dirname(os.path.dirname(__file__)))
```

Comment:
Um I just wanted to go two step back. Maybe there exists better approach? `os.path.abspath("../..")` maybe? I am not sure. :smile: